examples/generic-azure.yaml

```diff
@@ -164,12 +164,37 @@ groups:
             - flag: "udpu"
         remediation: |
           ## Remediation
-          It will be difficult to change the corosync MCAST transport to UCAST
-          on an existing cluster. Besides adding the 'transport: udpu' parameter one
-          would have to configure each node separatelly. The easiest way to
-          make the cluster UCAST it to bootstrap it. When creating the cluster
-          you would have to add the key -u, for example
-          $ crm cluster init -u ...
+          To change the corosync MCAST transport to UCAST edit the /etc/corosync/corosync.conf
+          as in the example
+          ```
+              max_messages: 20
+              interface {
+                  ringnumber: 0
+          -       bindnetaddr: 10.162.32.167
+          -       mcastaddr: 239.11.100.41
+                  mcastport: 5405
+                  ttl: 1
+              }
+          +   transport: udpu
+          ...
+          +nodelist {
+          +       node {
+          +               ring0_addr: 10.162.32.167
+          +               nodeid: 1
+          +       }
+          +
+          +       node {
+          +               ring0_addr: 10.162.32.89
+          +               nodeid: 2
+          +       }
+          +
+          +}
+          ```
+          1) In the totem section, in the interface subsection remove the
```

Comment:
Reading the HANA HA setup guide, it seems the first step is  "stop the already running cluster by using systemctl stop pacemaker" ? See section 9.1.3 of https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/